vc_zoom/indico_vc_zoom/api/client.py

```diff
@@ -0,0 +1,232 @@
+        return self.session.post(
+            "{}/users/{}/meetings".format(self.base_uri, user_id),
+            json=kwargs
+        )
+
+    def get(self, meeting_id, **kwargs):
+        return self.session.get("{}/meetings/{}".format(self.base_uri, meeting_id), json=kwargs)
+
+    def update(self, meeting_id, **kwargs):
+        if kwargs.get("start_time"):
+            kwargs["start_time"] = format_iso_dt(kwargs["start_time"])
+        return self.session.patch(
+            "{}/meetings/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+    def delete(self, meeting_id, **kwargs):
+        return self.session.delete(
+            "{}/meetings/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+
+class WebinarComponent(BaseComponent):
+    def list(self, user_id, **kwargs):
+        return self.get(
+            "{}/users/{}/webinars".format(self.base_uri, user_id), params=kwargs
+        )
+
+    def create(self, user_id, **kwargs):
+        if kwargs.get("start_time"):
+            kwargs["start_time"] = format_iso_dt(kwargs["start_time"])
+        return self.session.post(
+            "{}/users/{}/webinars".format(self.base_uri, user_id),
+            json=kwargs
+        )
+
+    def get(self, meeting_id, **kwargs):
+        return self.session.get("{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs)
+
+    def update(self, meeting_id, **kwargs):
+        if kwargs.get("start_time"):
+            kwargs["start_time"] = format_iso_dt(kwargs["start_time"])
+        return self.session.patch(
+            "{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+    def delete(self, meeting_id, **kwargs):
+        return self.session.delete(
+            "{}/webinars/{}".format(self.base_uri, meeting_id), json=kwargs
+        )
+
+
+class UserComponent(BaseComponent):
+    def me(self):
+        return self.get('me')
+
+    def list(self, **kwargs):
+        return self.session.get("{}/users".format(self.base_uri), params=kwargs)
+
+    def create(self, **kwargs):
+        return self.session.post("{}/users".format(self.base_uri), params=kwargs)
+
+    def update(self, user_id, **kwargs):
+        return self.session.patch("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+    def delete(self, user_id, **kwargs):
+        return self.session.delete("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+    def add_assistant(self, user_id, **kwargs):
+        return self.session.post("{}/users/{}/assistants".format(self.base_uri, user_id), json=kwargs)
+
+    def get_assistants(self, user_id, **kwargs):
+        return self.session.get("{}/users/{}/assistants".format(self.base_uri, user_id), params=kwargs)
+
+    def get(self, user_id, **kwargs):
+        return self.session.get("{}/users/{}".format(self.base_uri, user_id), params=kwargs)
+
+
+class ZoomClient(object):
+    """Zoom REST API Python Client."""
+
+    BASE_URI = "https://api.zoom.us/v2"
```

Comment:
Should we make this overridable? AFAIK zoom provides some kind of deeper on-premise integration where you'd have a custom API host.

OTOH, if someone needs that they could send a PR since it's not relevant for us atm..